docs/workflows.rst

```diff
@@ -43,6 +43,11 @@ migration.
     It is possible to re-run migration as many times as needed (if the Pulp 3 plugin which is
     being migrated is not used).
 
+.. warning::
```

Comment:
@goosemania please take a look and let me know if you have suggestions on the wording

Reply:
Based on the recent conversations we need to handle that case. Let's remove this warning then?